### ISSUE TITLE
[TE-demo] [Backend] H2 database schema is inconsistent with create-schema SQL script

### ISSUE THREAD

**Issue by jasonyanwenl** [apache/pinot#5913]
Hi team, 
when I tried to run TE in demo the first time in my local box it throws an error and I found the following two related issues for this:

* Current repo already contains a `thirdeye-pinot/config/h2db.mv.db` (check the file [here](https://github.com/apache/incubator-pinot/blob/master/thirdeye/thirdeye-pinot/config/h2db.mv.db)). However, the `init` method in `DaoProviderUtil` will check if such temporary db is existing. If so, it will not rerun `create-schema.sql`. The existing `h2db.mv.db` is pretty old and is not consistent with current `create-schema.sql`.

* Currently only when database is specified as h2 and **`h2db.mv.db` is not existing**, `create-schema.sql` is executed (check the code [here](https://github.com/apache/incubator-pinot/blob/331b874cd68b9a68b91d20b694bce63bcb5a5e2b/thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/datalayer/util/DaoProviderUtil.java#L100)). However, I guess this is not suitable because this means if a developer created a new table, he/she needs to manually delete the existing `h2db.mv.db` to force TE to rerun `create-schema.sql`?

So in conclusion, we have two questions:
* Should we remove the existing `thirdeye-pinot/config/h2db.mv.db`?
* Should we remove the check of existing `h2db.mv.db` in the `DaoProviderUtil::init()` ?

BTW, if this needs to be fixed, I could raise a PR. 

Thanks!

**Comment by Jackie-Jiang** [comment]
Closing third-eye related issue as the project has been moved out of the pinot repo. For unsolved issue that still applies, please create a new one under third-eye repo: https://github.com/project-thirdeye/thirdeye
